targets/UnrealMarketplacePlugin/templates/PlayFab/PlayFab_API.h.ejs

```diff
@@ -93,12 +93,16 @@ public:
 
     /** PlayFab Request Info */
     FString PlayFabRequestURL;
+<% if (api.name === "CloudScript") {
```

Comment:
This looks funky. Should the %> on line 97 really be at the end of line 96?

Reply:
For some reason, everywhere else in the SDK that's the pattern they use for if statements and hence why I followed.